session-name-1/slides/Makefile

```diff
@@ -0,0 +1,20 @@
+RVMD = reveal-md
+
+.PHONY: all html pdf clean
+
+all: html pdf
+
+html: _site
+
+_site: slides.md
+	$(RVMD) $< --static $@
+
+pdf: slides.pdf
+
+slides.pdf: slides.md
+	$(RVMD) $< --print $@
+
+clean:
+	-rm -f *~
+	-rm -fr _site
+	-rm -f slides.pdf
```

Comment:
For the same reason as above, I think you should move this to a generic Makefile that all other sessions can `include`.

Reply:
I'm still thinking whether it makes sense to simply build all session slides into a giant webpage. What do you think?

Reply:
What do you mean by "giant webpage"? Is it `_site/index.html`? If so, it's not that big (that's what she said). If you're referring to building all slides from all sessions into the same `index.html`, that's not what I meant. I suggested making the Makefile more generic and simply including it in the Makefile of each session.